lua/lspconfig/server_configurations/foam_ls.lua

```diff
@@ -0,0 +1,32 @@
+local util = require 'lspconfig.util'
+local bin_name = 'foam-ls'
+local cmd = { bin_name, '--stdio' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', bin_name, '--stdio' }
+end
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = { 'foam', 'OpenFOAM' },
+    root_dir = function(fname)
+      return util.search_ancestors(fname, function(path)
+        if util.path.exists(util.path.join(path, 'system', 'controlDict')) then
+          return path
+        end
+      end)
+    end,
+  },
+  docs = {
+    package_json = 'https://raw.githubusercontent.com/FoamScience/foam-language-server/master/package.json',
```

Comment:
I think this will cause some troubles still

Reply:
I thought we took it out yesterday 😭 ; I have:
```
"contributes": {
    "configuration": [
    ]
  }
```
so it should pass CI (like dartls does); but do you prefer another PR to remove it?

Reply:
I have a feeling these will be removed from lspconfig soon anyway.  Setting it to an array will at least make nvim-lsp-installer gracefully skip it :+1:. There are some other projects that rely on these containing a valid conf schema, I'm not sure how they'll behave (it'll prob all be fine)